_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: default
+title: Create reports with the Reporting CLI
+nav_order: 25
+---
+
+# Create reports with the Reporting CLI
+
+You can create Dashboard reports programmatically with the Reporting CLI without using the OpenSearch Dashboards and the Reporting plugin. This provides automation for reporting that can be connected to email workflows.
+
+For any Dashboard view you can request a report in CSV, PNG, or PDF format that is sent to an email address. This can be useful to send to multiple email recipients with an email alias.
```

Comment:
pdf, png can work with any webpage but for csv works only with discover page and limitations for csv are same as reporting plugin. Ref: https://opensearch.org/docs/latest/dashboards/reporting/